### PR TITLE
Add --dry-run-metadata

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: tar/multitape/multitape_write.c [hunk 1]
```diff
@@ -104,11 +104,11 @@ static int flushtape(TAPE_W *, int);
 /* Initialize stream. */
 static int
 stream_init(struct stream * S, chunkify_callback callback, void * cookie,
-    int no_chunkifier)
+    int use_chunkifier)
 {
 
 	/* Create chunkifier. */
-	if (!no_chunkifier) {
+	if (use_chunkifier) {
```
Comment (gperciva) [inline tar/multitape/multitape_write.c:111]: When I looked at the code again, I really didn't like the `(!no_chunkifier)` part.

**Comment by cperciva** [comment]
I prefer `no_chunkfier` as the variable name since it's the exceptional state -- it's basically a flag, and flags are things which flag non-default behaviour.

**Comment by gperciva** [comment]
Ok, I removed the REBASE commit.  As far as I know, this is ready for merging.

**Comment by cperciva** [comment]
Please fix `NEWS.md` conflict.

**Comment by gperciva** [comment]
Rebased.  Also fixed a comment in `writetape_open`, which would have been grammatically incorrect..

**Pull request merged**